pyqtgraph/examples/parametertree.py

```diff
@@ -149,5 +149,25 @@ def restore():
 compareState = p.saveState()
 assert pg.eq(compareState, state)
 
+
+
+## Create two ParameterTree widgets, both accessing the same data
```

Comment:
The additional lines added here are almost a copy of the lines above. On my system, the example above doesn't display, probably because the variables used by the example below overwrite those above. But in any case, I am not sure what functionality the new example is supposed to demonstrate that isn't already demonstrated by the one above?

Reply:
In fact there was not supposed to be here because now our example is in parameter_xml.py so I just put the initial file.